### DISCUSSION TITLE
KEDA Chart 2.5.1

### DISCUSSION THREAD

**Discussion by tomkerkhove** [kedacore/charts#220] [Announcements]
*Chart: v2.5.1 | App: v2.5.0*

## TL;DR
```shell
helm repo add keda https://kedacore.github.io/charts
helm install keda/keda
```

### New

- None

### Improvements

- Fix missing `list` & `watch` permissions for service account in cluster role (#216)
- Deploy Helm charts on N+5 different Kubernetes versions in CI (#213)

### Deprecations

- None

### Breaking Changes

- None

### Other

- None

## New Contributors
* @tomasspi made their first contribution in https://github.com/kedacore/charts/pull/216

**Full Changelog**: https://github.com/kedacore/charts/compare/v2.5.0...v2.5.1

<hr /><em>This discussion was created from the release <a href='https://github.com/kedacore/charts/releases/tag/v2.5.1'>KEDA Chart 2.5.1</a>.</em>
